certification/formatters/util.go

```diff
@@ -1,43 +1,55 @@
 package formatters
 
 import (
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/version"
 )
 
 // getResponse will extract the runtime's results and format it to fit the
 // UserResponse definition in a way that can then be formatted.
-func getResponse(r runtime.Results) runtime.UserResponse {
-	passedChecks := make([]certification.Metadata, len(r.Passed))
-	failedChecks := make([]certification.CheckInfo, len(r.Failed))
-	erroredChecks := make([]certification.HelpText, len(r.Errors))
+func getResponse(r runtime.Results) UserResponse {
+	passedChecks := make([]checkExecutionInfo, len(r.Passed))
+	failedChecks := make([]checkExecutionInfo, len(r.Failed))
+	erroredChecks := make([]checkExecutionInfo, len(r.Errors))
 
 	if len(r.Passed) > 0 {
 		for i, check := range r.Passed {
-			passedChecks[i] = check.Metadata()
+			passedChecks[i] = checkExecutionInfo{
+				Name:        check.Name(),
+				ElapsedTime: check.ElapsedTime.String(),
+				Description: check.Metadata().Description,
+			}
 		}
 	}
 
 	if len(r.Failed) > 0 {
 		for i, check := range r.Failed {
-			failedChecks[i] = certification.CheckInfo{
-				Metadata: check.Metadata(),
-				HelpText: check.Help(),
+			failedChecks[i] = checkExecutionInfo{
+				Name:        check.Name(),
+				ElapsedTime: check.ElapsedTime.String(),
```

Comment:
I suggest we limit the number of digits after the decimal point to two and round it to the nearest number.

Reply:
@samira-barouti I 100% agree, but I'm going to leave it as is for now and we can sort this out when we have our schema definition sorted. The reason here being that the Round and Truncate methods on time.Duration don't seem to do what I want when I read through their description, which means I'd have to convert to common unit (ms) and type (float64), round/truncate mathematically, and then reconvert back to the original unit. For now I'm thinking it's not worth the effort, but I definitely agree, and actually would prefer to have the same unit across the board in the final results anyway. 

We can definitely revist in the future.

Reply:
Fair enough :) Thanks for your insight!